### ISSUE TITLE
mount permission denied when running install

### ISSUE THREAD

**Issue by justechn** [gitsense/insight#12]
I am trying to install gitsense using the instructions found [here](https://gitsense.com/download/)
When I run ./install I get a message about executing `su gitsense -c "mkdir -p /tmp/gitsense-sudo-test; sudo mount -t tmpfs -o size=256K tmpfs /tmp/gitsense-sudo-test" 2>&1` but it fails.

So I broke the command apart to see where it is failing. When I run the mount command I get an error that says `mount: only root can use "--options" option`

Should i modify the install script to go back to the root user to run this command?

**Comment by gitsense** [comment]
Yes, you'll have to run the install as root.   If sudo is not setup for the gitsense account (which it wouldn't), the install script will update the system to grant the gitsense user mount permission.  GitSense creates tmpfs mounts (in memory filesystems), which is why the sudo setup is required.

Once the install has finished, everything runs as the gitsense user for security reason.

On a side note, you may want to install the latest release of GitSense as well.  I'm currently working on the install documentation https://gitsense.com/download/index.draft.html.  Note the links are currently broken but the videos work.

I'm hoping for next week to have general availability for the next GitSense release which contains massive changes/bug fixes/etc.

**Comment by gitsense** [comment]
I forgot to mention the version of GitSense in the Google WebStore will not work properly with the next version of GitSense.  I'm planning on updating the webstore version when GitSense becomes general available, but I can push the changes to the GitHub repo if you are interested in trying it out sooner.

The next release of GitSense works in standalone mode, so the chrome extension will no longer be a requirement.

**Comment by justechn** [comment]
I will try with the latest version, any chance to get a docker version? that would make installing it much easier.

**Comment by gitsense** [comment]
Yeah I would like to support docker, but I only have a high overview of how it works.  From what I've read so far, it seems like it would be pretty straightforward to make a docker version.
